pkg/sql/sem/builtins/builtins.go

```diff
@@ -2310,6 +2314,19 @@ var jsonTypeOfImpl = tree.Builtin{
 	Info: "Returns the type of the outermost JSON value as a text string.",
 }
 
+var toJSONImpl = tree.Builtin{
+	Types:      tree.ArgTypes{{"val", types.Any}},
+	ReturnType: tree.FixedReturnType(types.JSON),
+	Fn: func(_ *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+		j, err := args[0].AsJSON()
+		if err != nil {
+			return nil, err
+		}
+		return tree.NewDJSON(j), nil
+	},
+	Info: "Returns the value as json or jsonb.",
```

Comment:
nit: I think we usually capitalize type names in `Info`, though I don't think we're super consistent about it.